examples/01_plotting/plot_demo_plotting.py

```diff
@@ -49,8 +49,18 @@
     stat_img, threshold=3, title="plot_stat_map", cut_coords=[36, -27, 66]
 )
 
+# It's also possible to visualize volumes in a LR-flipped "radiological" view
+# Just set radiological=True
+plotting.plot_stat_map(
+    stat_img,
+    threshold=3,
+    title="plot_stat_map",
+    cut_coords=[36, -27, 66],
+    radiological=True,
+)
```

Comment:
Not really related to this PR but this is just moving this plot from the bottom of this example to here as it makes more sense to compare directly with the same plot above it but in radiological view